app/helpers/application_helper/button/reload_server_tree.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::ReloadServerTree < ApplicationHelper::Button::Basic
```

Comment:
No button has been assigned to this class. The assignment needs to be added.

/cc @romanblanco